debian/build.go

```diff
@@ -93,14 +94,23 @@ func (c cfg) run() error {
 		}
 	}
 
-	cmd := exec.Command("dpkg-buildpackage", "-us", "-uc", "-b")
+	cmd := exec.Command("dpkg-buildpackage", "-us", "-uc", "-b", "--host-arch=" + c.DebArch)
 	cmd.Dir = dstdir
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	if err := cmd.Run(); err != nil {
 		return err
 	}
 
+	os.MkdirAll("bin", os.ModeDir)
+
+	cp := exec.Command("cp", fmt.Sprintf("/tmp/%s_%s-%s_%s.deb", c.Package, c.Version, c.Revision, c.DebArch), fmt.Sprintf("bin/%s_%s-%s_%s.deb", c.Package, c.Version, c.Revision, c.DebArch))
```

Comment:
Can we just mv? Eventually we probably want to split up stable and unstable into seperate directories in bin/ as well